flash_image/flash_mfi.rst

```diff
@@ -3,4 +3,70 @@
 How to flash MFI
 ################
 
-TODO
+1. Parepare mfi image on your host computer.
+--------------------------------------------
+
+i) Download mfi image.  
+
+ a.   Go to the `device web <https://www.adlinktech.com/Products/ROS2_Solution/ROS2_Controller/ROScube-X?lang=en/>`_.  
+ b.  Login or create an account.  
+ c. Click BSP to download the BSP MFI Binary  
```

Comment:
Add the picture to show where is BSP in the website.

Reply:
Add two pictures.